packages/quantic/force-app/custom-token/main/default/classes/CustomTokenProvider.cls

```diff
@@ -0,0 +1,108 @@
+global with sharing class CustomTokenProvider implements ITokenProvider {
```

Comment:
Please add comments on functions that explains what they do, and a class comment to explain what this class does and why someone would use it vs. the package version etc etc

Remember, this is meant to be sent to devs, so they need to understand what this code does.

Reply:
I did comment on the function that I think needed it.

Please comment on each and every function signature that you deems unclear enough that it requires a comment.